firmware/include/widgets/weatherWidget.h

```diff
@@ -11,17 +11,18 @@
 
 #include "model/weatherDataModel.h"
 
-class WeatherWidget : public Widget {
-   public:
-    WeatherWidget(ScreenManager& manager);
+class WeatherWidget : public Widget
```

Comment:
I would suggest to refrain from formatting changes, because this will cause merge conflicts for everyone else working on the same files.